### ISSUE TITLE
qt4-base-mac fails to build on 10.14

### ISSUE THREAD

**Issue by kamischi** [fink/fink-distributions#323]
The comparison `if [[ $XCODE_VERSION > 8.9 ]]` in the patch script fails and the patch with `PatchFile7` is not performed, resulting in a build failure with the file `qfontengine_coretext.mm`. $XCODE_VERSION is 10.1 and the string comparison fails. No idea, how to perform a comparison with floating numbers.

**Comment by nieder** [comment]
I think any xcode >= 9 should work. Could test >= 9, but I'm trying to remember why I tested > 8.9 instead. Could cut $XCODE_VERSION to $XCODE_MAJOR_VERSION and then do an integer test on -ge 9. Not in a position to test right now.

**Comment by nieder** [comment]
I think this should work:
````
# Xcode 9 fix
export XCODE_MAJOR_VERSION=`printenv XCODE_VERSION | cut -d. -f1`
if [[ $XCODE_MAJOR_VERSION -ge 9 ]]; then
````

**Comment by nieder** [comment]
@kamischi can you try the code I pasted above as a replacement into the current .info and see if it works on 10.14/Xcode10? I tested it locally by setting dummy variables and it seemed to pass, but I want a test against a real install. Thanks.

**Comment by kamischi** [comment]
@nieder: The patch from above works. Tested on macOS 10.14.2 and Xcode 10.1.

**Comment by nieder** [comment]
Fixed by ca92be4
